static/js/settings_emoji.js

```diff
@@ -21,6 +21,15 @@ const meta = {
     loaded: false,
 };
 
+export function is_custom_emoji(emoji_name) {
+    const emoji_data = emoji.get_server_realm_emoji_data();
+    for (const emoji of Object.values(emoji_data)) {
+        if (emoji.name === emoji_name && !emoji.deactivated) {
+            return true;
+        }
+    }
+    return false;
+}
```

Comment:
Can you add an whiteline here before the declaration of `can_add_emoji` function. It's always good to have a look at other functions.

Reply:
I added a white line.